goaviatrix/aws_tgw_vpc_attachment.go

```diff
@@ -170,6 +171,14 @@ func (c *Client) GetAwsTgwVpcAttachment(awsTgwVpcAttachment *AwsTgwVpcAttachment
 		aTVA.CustomizedRouteAdvertisement = ""
 	}
 
+	firenetManagementDetails, err := c.GetFirenetManagementDetails(awsTgwVpcAttachment)
+	if err != nil {
+		return nil, errors.New("could not get firenet management details: " + err.Error())
+	}
+	if len(firenetManagementDetails) != 0 {
+		aTVA.EdgeAttachment = firenetManagementDetails[1]
```

Comment:
Why the `1` index not `0`? Will this slice always have >=2 values in it?

Reply:
`0` is the security domain name, `1` is the attachment name.